remotehandler.py

```
@@ -0,0 +1,294 @@
+import hashlib
+import os
+import sys
+import threading
+import datetime
+import base64
```

Ruff (0.8.2)

`base64` imported but unused

Remove unused import: `base64`

(F401)